### PR TITLE
Update Switch Gradle dependencies to latest versions table

### PR DESCRIPTION
**Pull request by cacheung**
Update the column title Extension name to SDK Name 
Update dependencies extensions name to match the gradle dependencies name 
Update the order of the list.  (Move edge to earlier of the list)

latestVersionTable

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/migrate-to-android.md [hunk 1]
```diff
@@ -16,18 +16,21 @@ At this time, the previous versions of the Android Mobile SDKs may be switched o
...
+| campaignclassic | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/campaignclassic.svg?logo=android&logoColor=white&label=campaignclassic)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/campaignclassic)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
 
 ### Manual library import
```
Comment (emdobrin) [inline src/pages/documentation/migrate-to-android.md:35]: the changes look good to me. aside from that, should we also include all the extensions to the gradle example below?

**Review by yangyansong-adbe — APPROVED** [review]

File: src/pages/documentation/migrate-to-android.md [hunk 2]
```diff
@@ -16,18 +16,21 @@ At this time, the previous versions of the Android Mobile SDKs may be switched o
...
+| assurance       | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/assurance.svg?logo=android&logoColor=white&label=assurance)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/assurance)    |
+| edge       | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/edge.svg?logo=android&logoColor=white&label=edge)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/edge) 
+| edgeidentity       | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/edgeidentity.svg?logo=android&logoColor=white&label=edgeidentity)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/edgeidentity) 
+| edgeconsent       | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/edgeconsent.svg?logo=android&logoColor=white&label=edgeconsent)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/edgeconsent)
```
Comment (emdobrin) [inline src/pages/documentation/migrate-to-android.md:26]: can you also include the messaging and optimize to the list?

Comment (swarna04) [reply src/pages/documentation/migrate-to-android.md:26]: Looks like we decided to keep all the SDK extensions. Following up on Emilia's comment here - please include messaging and optimize after Edge extensions but before the direct solution extensions.

Comment (cacheung) [reply src/pages/documentation/migrate-to-android.md:26]: Updated.

**Review by swarna04 — APPROVED** [review]

**Pull request merged**